src/EditorServicesCommandSuite/resources/SettingsStrings.resx

```diff
@@ -144,6 +144,15 @@
   <data name="CommandSplatRefactor_NewLineAfterHashtableDescription" xml:space="preserve">
     <value>Specifies whether a new line should be placed between the hashtable and the original command expression.</value>
   </data>
+  <data name="CommandSplatRefactor_AllParametersDescription" xml:space="preserve">
+    <value>Specifies whether all possible (remaining) parameters should be retrieved and added to the splat expression. The default behavior is to only convert currently bound parameters to a splat expression. Setting this to $null will enforce the default behavior.</value>
```

Comment:
I'm honestly blown away you thought to do this without even so much as a hint from me.  And figured out where to do it and what format. Bravo man, way to go the extra mile :)

Also change

`The default behavior is to only convert currently bound parameters to a splat expression.`

to

`The default behavior is to only convert arguments present in the original command expression.`

And `Setting this to $null will enforce the default behavior.` can be dropped.  I know that I was something I had something, but now that `New-CommandSuiteSettingsFile` writes the value from `DefaultFromSettings.Default` to the settings file, this is now implied.